eclipse-collections-code-generator/src/main/resources/impl/stack/immutable/immutablePrimitiveArrayStack.stg

```diff
@@ -180,6 +181,12 @@ final class Immutable<name>ArrayStack extends Abstract<name>Stack
         return new Immutable<name>StackSerializationProxy(this);
     }
 
+    @Override
+    public Mutable<name>Stack toStack()
+    {
+        return <name>Stacks.mutable.withAll(this);
```

Comment:
This should be `<name>Stacks.mutable.withAllReversed(this)` to be consistent with the object implementation.